### PR TITLE
Remove partner pod dead code and doc update

### PR DESCRIPTION
**Pull request by edcdavid**
This is to remove the unused partner/orchestrator code in the tnf suite. The partner repo is still required to instantiate the debug pods.
The documentation is also updated to reflect the new mechanisms

### CODE REVIEW

File: Dockerfile
Comment (javierpena) [inline Dockerfile:43]: I guess these changes will be reverted before merging, right? ;)

File: README.md
Comment (javierpena) [inline README.md:293]: typo: "binariy" vs "binary"

File: Dockerfile
Comment (hamadise) [reply Dockerfile:43]: right, this is used to allow .github CI to pass without issues.

File: README.md
Comment (hamadise) [reply README.md]: fixed.

File: Dockerfile
Comment (jun-rh) [inline Dockerfile:48]: this is not needed?

File: README.md
Comment (jun-rh) [inline README.md:293]: This paragraph should be removed now

File: README.md
Comment (jun-rh) [inline README.md:525]: And this

Comment (hamadise) [reply README.md:525]: fixed

File: README.md
Comment (hamadise) [reply README.md:293]: fixed

File: Dockerfile
Comment (hamadise) [reply Dockerfile:48]: before this change, we did not specify the branch, I guess it's defined as parameter when building the image, we can revert this change as well

Comment (hamadise) [reply Dockerfile:48]: what do you think ?

Comment (hamadise) [reply Dockerfile:48]: anyway, I'll wait for CI to pass. then remove it again and see the results.

**Review by hamadise — APPROVED** [review]

**Review by jun-rh — APPROVED** [review]

**Pull request merged**
